src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -144,6 +144,7 @@ public CftQueryService(CamundaService camundaService, CFTTaskMapper cftTaskMappe
             .filter(result -> result.containsKey(TASK_TYPE.value()))
             .map(result -> camundaService.getVariableValue(result.get(TASK_TYPE.value()), String.class))
             .filter(Objects::nonNull)
+            .filter(val -> val.trim().length() != 0)
```

Comment:
I believe the getVariableValue() attempts to read the value into an optional and it can either return a variable value or a null. 

Not sure if we need to do this length check. was there a particular case e.g a value was empty ?

Reply:
I was expecting that question :). Actually, pitest mutated with "" as response from DMN since we are not handling I had to filter the dmn response based on the "" and return empty results.

Reply:
Interesting I did not know about that in that case should we filter it once only with something like `StringUtils.hasText()` ?